guidelines/program-guidelines.md

```diff
@@ -0,0 +1,86 @@
+
+Overall, be consistent with how folks are introduced.
+
+## Logistics and Announcements
+
+A website or central document that describes the general practices and layout of the event is very valuable and can eliminate a lot of redundant questions.
+
+Try to keep all the information for the event in a central and logically organized manner. Keep in mind that weird things *will* happen so have a plan for emergency/urgent announcements (anyone have an HDMI cable?) and make sure people know where to look for these. Use the same medium (twitter/Slack/IRC/etc) as inconsistency will lead to confusion or missed announcements.
+
+Visit the venue at least once before the event, and prep the volunteers on the basics (especially if they are also coming in from out of town).
+
+## Things take time
+
+Although we “know” that things take time, we rarely schedule time for all the things, *beyond talks*,  that need to happen during an event.  Some of these things include:
+
+  - Applause (seriously)
+  - Plugging/Unplugging Laptops
+  - Speaker Introductions
+  - Announcements
+  - Sponsor talks (and other house-keeping)
+
+In order to realistically design your program, you should specifically set aside time to account for these things. We recommend:
+
+  - 10min at the beginning and end of the program day
+  - 5-10min between talks
+
+These times serve 2 purposes:
+
+  - realistically set time aside for things that have to happen and things **take time**
+  - You can use this time to recover speakers going over and under!
+
+You should not only set this time aside but you should clearly show it on the program you display and distribute to attendees. They appreciate a realistic schedule and will be more likely to comply with it if they know when it is and why it’s there in advance which always makes things run more smoothly.
+
+## Tech Check
+It is extremely useful to have your A/V or organizers available to help speakers set up their machines on stage to test their display *before* they have to speak. Inevitably, not every speaker will be available (and some will flake out and forget) but every person you are able to tech check saves you time in your program and also allows the event to appear more professional.
+
+We are all developers for the most part, and are aware of how computers are terrible and never work. Nevertheless we are an unforgiving group when it appears that people have not prepared for or are surprised that computers.
+
+Checking on equipment during breaks (coffee, lunch, etc) is a good way to make sure the equipment is still functional, or a good time for repairs.
+
+## Q&A
+We don’t recommend opening the floor to Q&A. We do recommend:
+
+  - Dedicated room/Speaker Corners
+  - Keep it short, speakers want to go to talks too!
```

Comment:
I agree that most Q&A's are bad, but I don't know if I fully agree with the recommendation. I don't know about how to phrase it better :/.

There's also moderated/submitted Q&A, where questions have to be submitted to the MC, who will ask them (or not). This also ensures that the Q&A is no essays in themselves, as people cannot submit more then 2-3 sentences.

Reply:
Yeah, we should rephrase that to list the options instead of excluding Q&A in total. I will think about the phrasing later.